brainiak/fcma/voxelselector.py

```diff
@@ -106,7 +106,7 @@ def __init__(self,
                  raw_data,
                  raw_data2=None,
                  voxel_unit=64,
-                 process_num=4,
+                 process_num=None,
```

Comment:
The `process_num` was intentionally set to be 4, a small number, instead of the "usable cpus" in the system because on a cluster node, the "usable cpus" could be over 30, which will likely blow up the memory for having the process duplicate so many copies.

Reply:
The memory for forked processes is allocated copy-on-write. Unless `_cross_validation_for_one_voxel` allocates a lot of data, it should be fine to have lots of processes in the pool.

Reply:
I understand this. Previously we observed memory blowout when `process_num` was set to be "usable cpus". So I would say yes...